src/client.ts

```diff
@@ -594,6 +621,8 @@ export class Client extends EventEmitter {
       const eventArgs: IClientErrorEventArgs = { kind: ClientEvent.Error, url, error }
       this.emit(ClientEvent.Error, eventArgs)
     } else if (result) {
+      console.log('event', event.id, result)
```

Comment:
If you want to leave this in change it to use `debug` instead of `console.log`.

Reply:
Ops I forgot this one